### ISSUE TITLE
Can't install VIB  - failed to validate content

### ISSUE THREAD

**Issue by GabryJay** [lamw/ghettoVCB#64]
Hi there!
I've 2 free HP esxi 5.5 servers. On the main one I installed months ago ghettoVCB with the old method.
Now I tried to install the VIB in the second server, but i get this error: 

VIB virtuallyGhetto_bootbank_ghettoVCB_1.0.0-0.0.0 violates extensibility rule checks: [u'(line 23: col 0) Element vib failed to validate content']

Is there a solution? Otherwise I'll proceed installing with the zip files as usual.
Thank you in advance!

**Comment by lamw** [comment]
Did you change the Acceptance Level to "Community" as Custom VIBs require lowering that to be able to install? You can always take a look at the logs to see what the issue is under /var/log/esxupdate.log

**Comment by GabryJay** [comment]
Sure, I changed the acceptance level after the first error message (which also contained the one I reported). Launching the installation the second time I received only the error above.
Here's the logs below: I read it yesterday but after reading and googling I didn't manage to find a solution, that's why I asked here :)

[can't upload a file here, so I paste the last lines]

2015-10-16T07:44:18Z esxupdate: vmware.runcommand: INFO: runcommand called with: args = '['/sbin/esxcfg-advcfg', '-q', '-g', '/UserVars/EsximageNetTimeout']', outfile = 'None', returnoutput = 'True', timeout = '0.0'.
2015-10-16T07:44:18Z esxupdate: vmware.runcommand: INFO: runcommand called with: args = '['/sbin/esxcfg-advcfg', '-q', '-g', '/UserVars/EsximageNetRetries']', outfile = 'None', returnoutput = 'True', timeout = '0.0'.
2015-10-16T07:44:18Z esxupdate: vmware.runcommand: INFO: runcommand called with: args = '['/sbin/esxcfg-advcfg', '-q', '-g', '/UserVars/EsximageNetRateLimit']', outfile = 'None', returnoutput = 'True', timeout = '0.0'.
2015-10-16T07:44:18Z esxupdate: root: INFO: Command = vib.install
2015-10-16T07:44:18Z esxupdate: root: INFO: Options = {'profile': None, 'nosigcheck': False, 'force': False, 'level': None, 'nomaintmode': False, 'downgrade': None, 'updateonly': False, 'proxy': None, 'viburl': ['/vmfs/volumes/datastore1/vghetto-ghettoVCB.vib'], 'dryrun': False, 'depot': None, 'nameid': None, 'noliveinstall': False, 'pending': None, 'oktoremove': False}
2015-10-16T07:44:18Z esxupdate: BootBankInstaller.pyc: INFO: Unrecognized value "title=Loading VMware ESXi" in boot.cfg
2015-10-16T07:44:18Z esxupdate: BootBankInstaller.pyc: INFO: Unrecognized value "title=Loading VMware ESXi" in boot.cfg
2015-10-16T07:44:18Z esxupdate: vmware.runcommand: INFO: runcommand called with: args = '['/sbin/bootOption', '-rp']', outfile = 'None', returnoutput = 'True', timeout = '0.0'.
2015-10-16T07:44:18Z esxupdate: vmware.runcommand: INFO: runcommand called with: args = '['/sbin/bootOption', '-ro']', outfile = 'None', returnoutput = 'True', timeout = '0.0'.
2015-10-16T07:44:18Z esxupdate: imageprofile: INFO: Adding VIB VMware_locker_tools-light_5.5.0-2.33.2068190 to ImageProfile HP-ESXi-5.5.0-Update2-iso-5.77.3
2015-10-16T07:44:18Z esxupdate: vmware.runcommand: INFO: runcommand called with: args = '['/sbin/esxcfg-advcfg', '-U', 'host-acceptance-level', '-G']', outfile = 'None', returnoutput = 'True', timeout = '0.0'.
2015-10-16T07:44:18Z esxupdate: Transaction: INFO: Skipping installed VIBs
2015-10-16T07:44:18Z esxupdate: Transaction: INFO: Final list of VIBs being installed: virtuallyGhetto_bootbank_ghettoVCB_1.0.0-0.0.0
2015-10-16T07:44:18Z esxupdate: imageprofile: INFO: Adding VIB virtuallyGhetto_bootbank_ghettoVCB_1.0.0-0.0.0 to ImageProfile HP-ESXi-5.5.0-Update2-iso-5.77.3
2015-10-16T07:44:18Z esxupdate: root: ERROR: Traceback (most recent call last):
2015-10-16T07:44:18Z esxupdate: root: ERROR:   File "/usr/lib/vmware/esxcli-software", line 441, in <module>
2015-10-16T07:44:18Z esxupdate: root: ERROR:     main()
2015-10-16T07:44:18Z esxupdate: root: ERROR:   File "/usr/lib/vmware/esxcli-software", line 432, in main
2015-10-16T07:44:18Z esxupdate: root: ERROR:     ret = CMDTABLE[command](options)
2015-10-16T07:44:18Z esxupdate: root: ERROR:   File "/usr/lib/vmware/esxcli-software", line 341, in VibInstallCmd
2015-10-16T07:44:18Z esxupdate: root: ERROR:     checkacceptance=checkacceptance)
2015-10-16T07:44:18Z esxupdate: root: ERROR:   File "/build/mts/release/bora-2068190/bora/build/esx/release/vmvisor/sys-boot/lib/python2.6/site-packages/vmware/esximage/Transaction.py", line 245, in InstallVibsFromSources
2015-10-16T07:44:18Z esxupdate: root: ERROR:   File "/build/mts/release/bora-2068190/bora/build/esx/release/vmvisor/sys-boot/lib/python2.6/site-packages/vmware/esximage/Transaction.py", line 347, in _installVibs
2015-10-16T07:44:18Z esxupdate: root: ERROR:   File "/build/mts/release/bora-2068190/bora/build/esx/release/vmvisor/sys-boot/lib/python2.6/site-packages/vmware/esximage/Transaction.py", line 384, in _validateAndInstallProfile
2015-10-16T07:44:18Z esxupdate: root: ERROR: DependencyError: VIB virtuallyGhetto_bootbank_ghettoVCB_1.0.0-0.0.0 violates extensibility rule checks: [u'(line 23: col 0) Element vib failed to validate content']
